tests/sample/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
If we rename the examples test to executable test, then this change can be undone.

Reply:
I'd be happy to do this but I'm not sure which is clearer. "example" is probably more clear than "sample", true, but given we have other "example"s it's still a bit confusing. Does "starter" make sense? Or something similar?

Reply:
Not sure the context of 'starter' and what we are trying to say with that. I actually think that example and sample means the same thing here. However we have a Kinect Sample repo, and which has samples that may be incomplete or have dependencies that we don't want to bring in to the SDK project. So I like the idea of exclusively using "examples" to mean Azure-Kinect-Sensor-SDK repro and "samples" to mean the Azure Kinect Samples repo.

Reply:
Should be fixed